  INFO root [0x000019e0] [2019-03-02 09:58:20,736] - 
  INFO root [0x000019e0] [2019-03-02 09:58:20,737] - -----------------------------------------------------------------------------------------


  INFO root [0x000019e0] [2019-03-02 09:58:20,738] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [0x000019e0] [2019-03-02 09:58:20,738] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [0x000019e0] [2019-03-02 09:58:20,738] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [0x000019e0] [2019-03-02 09:58:20,742] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:20015 )
  INFO root [0x000019e0] [2019-03-02 09:58:20,742] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.0.148:20015, SOMAXCONN=128.
  INFO root [0x000019e0] [2019-03-02 09:58:20,745] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:20005 )
  INFO root [0x000019e0] [2019-03-02 09:58:20,745] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.0.148:20005, SOMAXCONN=128.
  INFO root [0x000019e0] [2019-03-02 09:58:20,748] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:13461 )
  INFO root [0x000019e0] [2019-03-02 09:58:20,748] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.148:13461, SOMAXCONN=128.
  INFO root [0x000019e0] [2019-03-02 09:58:20,748] - server-configs:
  INFO root [0x000019e0] [2019-03-02 09:58:20,748] - 	gameUpdateHertz : 10
  INFO root [0x000019e0] [2019-03-02 09:58:20,748] - 	entryScriptFile : kbemain
  INFO root [0x000019e0] [2019-03-02 09:58:20,748] - 	internalTcpAddr : 192.168.0.148:13461
  INFO root [0x000019e0] [2019-03-02 09:58:20,749] - 	externalTcpAddr : 192.168.0.148:20015
  INFO root [0x000019e0] [2019-03-02 09:58:20,749] - 	externalUdpAddr : 192.168.0.148:20005
  INFO root [0x000019e0] [2019-03-02 09:58:20,749] - 	componentID : 7001
  WARN root [0x000019e0] [2019-03-02 09:58:20,981] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [0x000019e0] [2019-03-02 09:58:21,012] - Components::findLogger: found logger, addr:192.168.0.148:13264
  INFO root [0x000019e0] [2019-03-02 09:58:21,013] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [0x000019e0] [2019-03-02 09:58:21,013] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000019e0] [2019-03-02 09:58:21,017] - ---- baseapp Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: 18914A7099A0127BD73DD63149F1D9BA. Config: Debug X86. Built: 17:20:50 Feb 27 2019. AppID: 7001. UID: 28936. PID: 11720 ----
  INFO root [0x000019e0] [2019-03-02 09:58:21,017] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [0x000019e0] [2019-03-02 09:58:21,017] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000019e0] [2019-03-02 09:58:21,018] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x000019e0] [2019-03-02 09:58:21,183] - Script::install(): is successfully, Python=(3.7.2 (default, Feb 27 2019, 17:13:46) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x000019e0] [2019-03-02 09:58:21,183] - Script::install(): KBExtra is successfully!
 DEBUG root [0x000019e0] [2019-03-02 09:58:21,481] - loaded entity-script:Account(1).
 DEBUG root [0x000019e0] [2019-03-02 09:58:21,481] - loaded entity-script:Halls(2).
 DEBUG root [0x000019e0] [2019-03-02 09:58:21,481] - loaded entity-script:Room(3).
  INFO root [0x000019e0] [2019-03-02 09:58:21,481] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x000019e0] [2019-03-02 09:58:21,482] - TelnetServer server is running on port 40000
  INFO root [0x000019e0] [2019-03-02 09:58:21,819] - ---- baseapp is running ----
 DEBUG root [0x000019e0] [2019-03-02 09:58:21,820] - Components::process(): Request for the process of identity...
  WARN root [0x000019e0] [2019-03-02 09:58:27,945] - Forwarding logs to logger[192.168.0.148:13264/0/0/0]...
  INFO root [0x00002240] [2019-03-02 10:02:26,667] - 
  INFO root [0x00002240] [2019-03-02 10:02:26,667] - -----------------------------------------------------------------------------------------


  INFO root [0x00002240] [2019-03-02 10:02:26,667] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [0x00002240] [2019-03-02 10:02:26,667] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [0x00002240] [2019-03-02 10:02:26,667] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [0x00002240] [2019-03-02 10:02:26,667] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:20015 )
  INFO root [0x00002240] [2019-03-02 10:02:26,667] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.0.148:20015, SOMAXCONN=128.
  INFO root [0x00002240] [2019-03-02 10:02:26,667] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:20005 )
  INFO root [0x00002240] [2019-03-02 10:02:26,667] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.0.148:20005, SOMAXCONN=128.
  INFO root [0x00002240] [2019-03-02 10:02:26,683] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:1732 )
  INFO root [0x00002240] [2019-03-02 10:02:26,683] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.148:1732, SOMAXCONN=128.
  INFO root [0x00002240] [2019-03-02 10:02:26,683] - server-configs:
  INFO root [0x00002240] [2019-03-02 10:02:26,683] - 	gameUpdateHertz : 10
  INFO root [0x00002240] [2019-03-02 10:02:26,683] - 	entryScriptFile : kbemain
  INFO root [0x00002240] [2019-03-02 10:02:26,683] - 	internalTcpAddr : 192.168.0.148:1732
  INFO root [0x00002240] [2019-03-02 10:02:26,683] - 	externalTcpAddr : 192.168.0.148:20015
  INFO root [0x00002240] [2019-03-02 10:02:26,683] - 	externalUdpAddr : 192.168.0.148:20005
  INFO root [0x00002240] [2019-03-02 10:02:26,683] - 	componentID : 7001
  WARN root [0x00002240] [2019-03-02 10:02:26,933] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [0x00002240] [2019-03-02 10:02:26,948] - ---- baseapp Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: 18914A7099A0127BD73DD63149F1D9BA. Config: Debug X86. Built: 17:20:50 Feb 27 2019. AppID: 7001. UID: 28936. PID: 11044 ----
  INFO root [0x00002240] [2019-03-02 10:02:26,948] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [0x00002240] [2019-03-02 10:02:26,948] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002240] [2019-03-02 10:02:26,948] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x00002240] [2019-03-02 10:02:27,120] - Script::install(): is successfully, Python=(3.7.2 (default, Feb 27 2019, 17:13:46) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x00002240] [2019-03-02 10:02:27,120] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00002240] [2019-03-02 10:02:27,417] - loaded entity-script:Account(1).
 DEBUG root [0x00002240] [2019-03-02 10:02:27,417] - loaded entity-script:Halls(2).
 DEBUG root [0x00002240] [2019-03-02 10:02:27,417] - loaded entity-script:Room(3).
  INFO root [0x00002240] [2019-03-02 10:02:27,417] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00002240] [2019-03-02 10:02:27,417] - TelnetServer server is running on port 40000
  INFO root [0x00002240] [2019-03-02 10:02:27,761] - ---- baseapp is running ----
 DEBUG root [0x00002240] [2019-03-02 10:02:27,761] - Components::process(): Request for the process of identity...
  WARN root [0x00002240] [2019-03-02 10:02:34,057] - EntityApp::checkTickPeriod: tick took 6281% (6.28 seconds)!
  INFO root [0x00002240] [2019-03-02 10:02:35,166] - Components::findComponents: find logger(1)...
  WARN root [0x00002240] [2019-03-02 10:02:36,244] - Components::findComponents: not found logger!
  WARN root [0x00002240] [2019-03-02 10:02:36,244] - EntityApp::checkTickPeriod: tick took 1172% (1.17 seconds)!
  INFO root [0x00002240] [2019-03-02 10:02:37,276] - Components::findComponents: find dbmgr(1)...
  WARN root [0x00002240] [2019-03-02 10:02:38,400] - EntityApp::checkTickPeriod: tick took 1234% (1.23 seconds)!
  INFO root [0x00002240] [2019-03-02 10:02:39,432] - Components::findComponents: find dbmgr(2)...
  WARN root [0x00002240] [2019-03-02 10:02:40,478] - EntityApp::checkTickPeriod: tick took 1110% (1.11 seconds)!
  INFO root [0x00002240] [2019-03-02 10:02:41,572] - Components::findComponents: find dbmgr(3)...
  WARN root [0x00002240] [2019-03-02 10:02:42,666] - EntityApp::checkTickPeriod: tick took 1187% (1.19 seconds)!
  INFO root [0x00002240] [2019-03-02 10:02:43,681] - Components::findComponents: find dbmgr(4)...
  WARN root [0x00002240] [2019-03-02 10:02:44,759] - EntityApp::checkTickPeriod: tick took 1093% (1.09 seconds)!
  INFO root [0x00002240] [2019-03-02 10:02:45,775] - Components::findComponents: find dbmgr(5)...
  WARN root [0x00002240] [2019-03-02 10:02:46,791] - EntityApp::checkTickPeriod: tick took 1125% (1.13 seconds)!
  INFO root [0x00002240] [2019-03-02 10:02:47,871] - Components::findComponents: find dbmgr(6)...
  WARN root [0x00002240] [2019-03-02 10:02:48,980] - EntityApp::checkTickPeriod: tick took 1203% (1.20 seconds)!
  INFO root [0x00002240] [2019-03-02 10:02:50,074] - Components::findComponents: find dbmgr(7)...
  WARN root [0x00002240] [2019-03-02 10:02:51,167] - EntityApp::checkTickPeriod: tick took 1203% (1.20 seconds)!
  INFO root [0x00002240] [2019-03-02 10:02:52,183] - Components::findComponents: find dbmgr(8)...
  WARN root [0x00002240] [2019-03-02 10:02:53,245] - EntityApp::checkTickPeriod: tick took 1078% (1.08 seconds)!
  INFO root [0x00002240] [2019-03-02 10:02:54,277] - Components::findComponents: find dbmgr(9)...
  WARN root [0x00002240] [2019-03-02 10:02:55,323] - EntityApp::checkTickPeriod: tick took 1156% (1.16 seconds)!
  INFO root [0x00002240] [2019-03-02 10:02:56,370] - Components::findComponents: find dbmgr(10)...
  WARN root [0x00002240] [2019-03-02 10:02:57,401] - EntityApp::checkTickPeriod: tick took 1125% (1.13 seconds)!
  INFO root [0x00002240] [2019-03-02 10:02:58,433] - Components::findComponents: find dbmgr(11)...
  WARN root [0x00002240] [2019-03-02 10:02:59,479] - EntityApp::checkTickPeriod: tick took 1109% (1.11 seconds)!
  INFO root [0x00002240] [2019-03-02 10:03:00,573] - Components::findComponents: find dbmgr(12)...
  WARN root [0x00002240] [2019-03-02 10:03:01,667] - EntityApp::checkTickPeriod: tick took 1187% (1.19 seconds)!
  INFO root [0x00002240] [2019-03-02 10:03:02,682] - Components::findComponents: find dbmgr(13)...
  WARN root [0x00002240] [2019-03-02 10:03:03,745] - EntityApp::checkTickPeriod: tick took 1078% (1.08 seconds)!
  INFO root [0x00002240] [2019-03-02 10:03:04,776] - Components::findComponents: find dbmgr(14)...
  WARN root [0x00002240] [2019-03-02 10:03:05,823] - EntityApp::checkTickPeriod: tick took 1156% (1.16 seconds)!
  INFO root [0x00002240] [2019-03-02 10:03:06,869] - Components::findComponents: find dbmgr(15)...
  WARN root [0x00002240] [2019-03-02 10:03:07,888] - EntityApp::checkTickPeriod: tick took 1112% (1.11 seconds)!
  INFO root [0x00002240] [2019-03-02 10:03:08,966] - Components::findComponents: find dbmgr(16)...
  WARN root [0x00002240] [2019-03-02 10:03:10,091] - EntityApp::checkTickPeriod: tick took 1219% (1.22 seconds)!
  WARN root [0x00002240] [2019-03-02 10:03:11,169] - Components::findComponents: find dbmgr(17)...
delay time is too long, please check the dbmgr logs!
  WARN root [0x00002240] [2019-03-02 10:03:12,278] - EntityApp::checkTickPeriod: tick took 1203% (1.20 seconds)!
  WARN root [0x00002240] [2019-03-02 10:03:13,372] - Components::findComponents: find dbmgr(18)...
delay time is too long, please check the dbmgr logs!
  WARN root [0x00002240] [2019-03-02 10:03:14,466] - EntityApp::checkTickPeriod: tick took 1187% (1.19 seconds)!
  WARN root [0x00002240] [2019-03-02 10:03:15,481] - Components::findComponents: find dbmgr(19)...
delay time is too long, please check the dbmgr logs!
  WARN root [0x00002240] [2019-03-02 10:03:16,544] - EntityApp::checkTickPeriod: tick took 1078% (1.08 seconds)!
  WARN root [0x00002240] [2019-03-02 10:03:17,575] - Components::findComponents: find dbmgr(20)...
delay time is too long, please check the dbmgr logs!
  WARN root [0x00002240] [2019-03-02 10:03:18,622] - EntityApp::checkTickPeriod: tick took 1156% (1.16 seconds)!
  WARN root [0x00002240] [2019-03-02 10:03:19,668] - Components::findComponents: find dbmgr(21)...
delay time is too long, please check the dbmgr logs!
  WARN root [0x00002240] [2019-03-02 10:03:20,700] - EntityApp::checkTickPeriod: tick took 1125% (1.13 seconds)!
  WARN root [0x00002240] [2019-03-02 10:03:21,778] - Components::findComponents: find dbmgr(22)...
delay time is too long, please check the dbmgr logs!
  WARN root [0x00002240] [2019-03-02 10:03:22,887] - EntityApp::checkTickPeriod: tick took 1218% (1.22 seconds)!
  INFO root [0x00002d04] [2019-03-02 10:03:39,058] - 
  INFO root [0x00002d04] [2019-03-02 10:03:39,058] - -----------------------------------------------------------------------------------------


  INFO root [0x00002d04] [2019-03-02 10:03:39,058] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [0x00002d04] [2019-03-02 10:03:39,058] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [0x00002d04] [2019-03-02 10:03:39,058] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [0x00002d04] [2019-03-02 10:03:39,073] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:20015 )
  INFO root [0x00002d04] [2019-03-02 10:03:39,073] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.0.148:20015, SOMAXCONN=128.
  INFO root [0x00002d04] [2019-03-02 10:03:39,073] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:20005 )
  INFO root [0x00002d04] [2019-03-02 10:03:39,073] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.0.148:20005, SOMAXCONN=128.
  INFO root [0x00002d04] [2019-03-02 10:03:39,073] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:1767 )
  INFO root [0x00002d04] [2019-03-02 10:03:39,089] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.148:1767, SOMAXCONN=128.
  INFO root [0x00002d04] [2019-03-02 10:03:39,089] - server-configs:
  INFO root [0x00002d04] [2019-03-02 10:03:39,089] - 	gameUpdateHertz : 10
  INFO root [0x00002d04] [2019-03-02 10:03:39,089] - 	entryScriptFile : kbemain
  INFO root [0x00002d04] [2019-03-02 10:03:39,089] - 	internalTcpAddr : 192.168.0.148:1767
  INFO root [0x00002d04] [2019-03-02 10:03:39,089] - 	externalTcpAddr : 192.168.0.148:20015
  INFO root [0x00002d04] [2019-03-02 10:03:39,089] - 	externalUdpAddr : 192.168.0.148:20005
  INFO root [0x00002d04] [2019-03-02 10:03:39,089] - 	componentID : 7001
  WARN root [0x00002d04] [2019-03-02 10:03:39,433] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [0x00002d04] [2019-03-02 10:03:39,683] - Components::findLogger: found logger, addr:192.168.0.148:1763
  INFO root [0x00002d04] [2019-03-02 10:03:39,683] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [0x00002d04] [2019-03-02 10:03:39,683] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002d04] [2019-03-02 10:03:39,698] - ---- baseapp Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: 18914A7099A0127BD73DD63149F1D9BA. Config: Debug X86. Built: 17:20:50 Feb 27 2019. AppID: 7001. UID: 28936. PID: 11800 ----
  INFO root [0x00002d04] [2019-03-02 10:03:39,698] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [0x00002d04] [2019-03-02 10:03:39,698] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002d04] [2019-03-02 10:03:39,698] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x00002d04] [2019-03-02 10:03:39,886] - Script::install(): is successfully, Python=(3.7.2 (default, Feb 27 2019, 17:13:46) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x00002d04] [2019-03-02 10:03:39,886] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00002d04] [2019-03-02 10:03:39,979] - loaded entity-script:Account(1).
 DEBUG root [0x00002d04] [2019-03-02 10:03:39,979] - loaded entity-script:Halls(2).
 DEBUG root [0x00002d04] [2019-03-02 10:03:39,979] - loaded entity-script:Room(3).
  INFO root [0x00002d04] [2019-03-02 10:03:39,979] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00002d04] [2019-03-02 10:03:39,979] - TelnetServer server is running on port 40000
  INFO root [0x00002d04] [2019-03-02 10:03:40,401] - ---- baseapp is running ----
 DEBUG root [0x00002d04] [2019-03-02 10:03:40,401] - Components::process(): Request for the process of identity...
  WARN root [0x00002d04] [2019-03-02 10:03:46,682] - Forwarding logs to logger[192.168.0.148:1763/0/0/0]...
 ERROR root [0x00002d04] [2019-03-02 10:09:01,266] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [0x00002d04] [2019-03-02 10:09:01,268] - The following logs sent to logger failed:
  WARN root [0x00002d04] [2019-03-02 10:09:01,268] - ==> [2019-03-02 10:09:01 265] TCPPacketReceiver::processPendingEvents(192.168.0.148:1763): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002d04] [2019-03-02 10:09:01,269] - ==> [2019-03-02 10:09:01 266] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [0x00002d04] [2019-03-02 10:09:01,269] - TCPPacketReceiver::processPendingEvents(192.168.0.148:1768): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002d04] [2019-03-02 10:09:01,270] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [0x00002d04] [2019-03-02 10:09:01,312] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00002d04] [2019-03-02 10:09:01,313] - onBaseAppShutDown: state=0
  INFO root [0x00002d04] [2019-03-02 10:09:01,313] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00002d04] [2019-03-02 10:09:01,313] - onBaseAppShutDown: state=1
  INFO root [0x00002d04] [2019-03-02 10:09:01,314] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00002d04] [2019-03-02 10:09:01,315] - onBaseAppShutDown: state=2
  WARN root [0x00002d04] [2019-03-02 10:09:01,316] - TCPPacketReceiver::processPendingEvents(192.168.0.148:1766): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [0x00002d04] [2019-03-02 10:09:01,316] - Components::removeComponentByChannel: baseappmgr : 5000, Normal exit!
  WARN root [0x00002d04] [2019-03-02 10:09:01,316] - ServerApp::shutDown:  In shuttingdown!
  WARN root [0x00002d04] [2019-03-02 10:09:01,316] - TCPPacketReceiver::processPendingEvents(192.168.0.148:1761): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [0x00002d04] [2019-03-02 10:09:01,316] - Components::removeComponentByChannel: cellappmgr : 6000, Normal exit!
  WARN root [0x00002d04] [2019-03-02 10:09:01,317] - TCPPacketReceiver::processPendingEvents(192.168.0.148:1764): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [0x00002d04] [2019-03-02 10:09:01,317] - Components::removeComponentByChannel: cellapp : 8001, Normal exit!
 ERROR root [0x00002d04] [2019-03-02 10:09:01,399] - Account::onCellWriteToDBCompleted(2): not found dbmgr!
 ERROR root [0x00002d04] [2019-03-02 10:09:01,400] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x00002d04] [2019-03-02 10:09:01,400] - Halls::~Halls(): 1
 DEBUG root [0x00002d04] [2019-03-02 10:09:01,400] - Account::~Account(): 2
 DEBUG root [0x00002d04] [2019-03-02 10:09:01,401] - Room::~Room(): 3
  INFO root [0x00002d04] [2019-03-02 10:09:01,445] - Script::uninstall(): is successfully!
 DEBUG root [0x00002d04] [2019-03-02 10:09:01,445] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00002d04] [2019-03-02 10:09:01,767] - ThreadPool::destroy(): waiting for thread(2)[0x1941df0,0x19415d0,], try=1
 DEBUG root [0x00002d04] [2019-03-02 10:09:02,169] - TPThread::~TPThread(): 0x1941df0
 DEBUG root [0x00002d04] [2019-03-02 10:09:02,169] - TPThread::~TPThread(): 0x19415d0
 DEBUG root [0x00002d04] [2019-03-02 10:09:02,169] - ThreadPool::destroy(): successfully!
 DEBUG root [0x00002d04] [2019-03-02 10:09:02,202] - Bundle::destroyObjPool(): size 5.
 DEBUG root [0x00002d04] [2019-03-02 10:09:02,203] - Channel::destroyObjPool(): size 15.
 DEBUG root [0x00002d04] [2019-03-02 10:09:02,203] - TCPPacket::destroyObjPool(): size 4.
 DEBUG root [0x00002d04] [2019-03-02 10:09:02,203] - UDPPacket::destroyObjPool(): size 16.
 DEBUG root [0x00002d04] [2019-03-02 10:09:02,203] - EndPoint::destroyObjPool(): size 15.
 DEBUG root [0x00002d04] [2019-03-02 10:09:02,204] - Address::destroyObjPool(): size 0.
 DEBUG root [0x00002d04] [2019-03-02 10:09:02,204] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [0x00002d04] [2019-03-02 10:09:02,204] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [0x00002d04] [2019-03-02 10:09:02,204] - baseapp(7001) has shut down.
  INFO root [0x00002d04] [2019-03-02 10:09:03,209] - EventDispatcher()::~EventDispatcher: Num timers = 1
  INFO root [0x00002348] [2019-03-02 10:10:38,190] - 
  INFO root [0x00002348] [2019-03-02 10:10:38,190] - -----------------------------------------------------------------------------------------


  INFO root [0x00002348] [2019-03-02 10:10:38,190] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [0x00002348] [2019-03-02 10:10:38,190] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [0x00002348] [2019-03-02 10:10:38,190] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [0x00002348] [2019-03-02 10:10:38,204] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:20015 )
  INFO root [0x00002348] [2019-03-02 10:10:38,204] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.0.123:20015, SOMAXCONN=128.
  INFO root [0x00002348] [2019-03-02 10:10:38,204] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:20005 )
  INFO root [0x00002348] [2019-03-02 10:10:38,204] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.0.123:20005, SOMAXCONN=128.
  INFO root [0x00002348] [2019-03-02 10:10:38,210] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:2715 )
  INFO root [0x00002348] [2019-03-02 10:10:38,210] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.123:2715, SOMAXCONN=128.
  INFO root [0x00002348] [2019-03-02 10:10:38,210] - server-configs:
  INFO root [0x00002348] [2019-03-02 10:10:38,210] - 	gameUpdateHertz : 10
  INFO root [0x00002348] [2019-03-02 10:10:38,210] - 	entryScriptFile : kbemain
  INFO root [0x00002348] [2019-03-02 10:10:38,210] - 	internalTcpAddr : 192.168.0.123:2715
  INFO root [0x00002348] [2019-03-02 10:10:38,210] - 	externalTcpAddr : 192.168.0.123:20015
  INFO root [0x00002348] [2019-03-02 10:10:38,210] - 	externalUdpAddr : 192.168.0.123:20005
  INFO root [0x00002348] [2019-03-02 10:10:38,210] - 	componentID : 7001
  WARN root [0x00002348] [2019-03-02 10:10:38,490] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [0x00002348] [2019-03-02 10:10:38,560] - Components::findLogger: found logger, addr:192.168.0.123:2683
  INFO root [0x00002348] [2019-03-02 10:10:38,560] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [0x00002348] [2019-03-02 10:10:38,560] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002348] [2019-03-02 10:10:38,570] - ---- baseapp Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: 18914A7099A0127BD73DD63149F1D9BA. Config: Debug X86. Built: 17:20:50 Feb 27 2019. AppID: 7001. UID: 28936. PID: 5628 ----
  INFO root [0x00002348] [2019-03-02 10:10:38,570] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [0x00002348] [2019-03-02 10:10:38,570] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002348] [2019-03-02 10:10:38,570] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x00002348] [2019-03-02 10:10:38,749] - Script::install(): is successfully, Python=(3.7.2 (default, Feb 27 2019, 17:13:46) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x00002348] [2019-03-02 10:10:38,749] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00002348] [2019-03-02 10:10:39,069] - loaded entity-script:Account(1).
 DEBUG root [0x00002348] [2019-03-02 10:10:39,069] - loaded entity-script:Halls(2).
 DEBUG root [0x00002348] [2019-03-02 10:10:39,069] - loaded entity-script:Room(3).
  INFO root [0x00002348] [2019-03-02 10:10:39,071] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00002348] [2019-03-02 10:10:39,071] - TelnetServer server is running on port 40000
  INFO root [0x00002348] [2019-03-02 10:10:39,470] - ---- baseapp is running ----
 DEBUG root [0x00002348] [2019-03-02 10:10:39,470] - Components::process(): Request for the process of identity...
  WARN root [0x00002348] [2019-03-02 10:10:45,779] - Forwarding logs to logger[192.168.0.123:2683/0/0/0]...
 ERROR root [0x00002348] [2019-03-02 10:13:34,109] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [0x00002348] [2019-03-02 10:13:34,109] - The following logs sent to logger failed:
  WARN root [0x00002348] [2019-03-02 10:13:34,109] - ==> [2019-03-02 10:13:34 112] TCPPacketReceiver::processPendingEvents(192.168.0.123:2683): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002348] [2019-03-02 10:13:34,109] - ==> [2019-03-02 10:13:34 113] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [0x00002348] [2019-03-02 10:13:34,139] - TCPPacketReceiver::processPendingEvents(192.168.0.123:2685): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002348] [2019-03-02 10:13:34,139] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [0x00002348] [2019-03-02 10:13:34,139] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00002348] [2019-03-02 10:13:34,149] - onBaseAppShutDown: state=0
  INFO root [0x00002348] [2019-03-02 10:13:34,149] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00002348] [2019-03-02 10:13:34,149] - onBaseAppShutDown: state=1
  INFO root [0x00002348] [2019-03-02 10:13:34,149] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00002348] [2019-03-02 10:13:34,149] - onBaseAppShutDown: state=2
 ERROR root [0x00002348] [2019-03-02 10:13:34,159] - Account::onCellWriteToDBCompleted(2): not found dbmgr!
 ERROR root [0x00002348] [2019-03-02 10:13:34,159] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00002348] [2019-03-02 10:13:34,159] - Account::onCellWriteToDBCompleted(4): not found dbmgr!
 ERROR root [0x00002348] [2019-03-02 10:13:34,169] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x00002348] [2019-03-02 10:13:34,169] - Halls::~Halls(): 1
 DEBUG root [0x00002348] [2019-03-02 10:13:34,169] - Account::~Account(): 2
 DEBUG root [0x00002348] [2019-03-02 10:13:34,169] - Account::~Account(): 4
 DEBUG root [0x00002348] [2019-03-02 10:13:34,169] - Room::~Room(): 3
  INFO root [0x00002348] [2019-03-02 10:13:34,169] - Script::uninstall(): is successfully!
 DEBUG root [0x00002348] [2019-03-02 10:13:34,169] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00000c20] [2019-03-02 10:16:41,242] - 
  INFO root [0x00000c20] [2019-03-02 10:16:41,249] - -----------------------------------------------------------------------------------------


  INFO root [0x00000c20] [2019-03-02 10:16:41,249] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [0x00000c20] [2019-03-02 10:16:41,249] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [0x00000c20] [2019-03-02 10:16:41,249] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [0x00000c20] [2019-03-02 10:16:41,259] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:20015 )
  INFO root [0x00000c20] [2019-03-02 10:16:41,259] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.0.123:20015, SOMAXCONN=128.
  INFO root [0x00000c20] [2019-03-02 10:16:41,263] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:20005 )
  INFO root [0x00000c20] [2019-03-02 10:16:41,263] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.0.123:20005, SOMAXCONN=128.
  INFO root [0x00000c20] [2019-03-02 10:16:41,263] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:2854 )
  INFO root [0x00000c20] [2019-03-02 10:16:41,263] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.123:2854, SOMAXCONN=128.
  INFO root [0x00000c20] [2019-03-02 10:16:41,263] - server-configs:
  INFO root [0x00000c20] [2019-03-02 10:16:41,263] - 	gameUpdateHertz : 10
  INFO root [0x00000c20] [2019-03-02 10:16:41,263] - 	entryScriptFile : kbemain
  INFO root [0x00000c20] [2019-03-02 10:16:41,263] - 	internalTcpAddr : 192.168.0.123:2854
  INFO root [0x00000c20] [2019-03-02 10:16:41,263] - 	externalTcpAddr : 192.168.0.123:20015
  INFO root [0x00000c20] [2019-03-02 10:16:41,263] - 	externalUdpAddr : 192.168.0.123:20005
  INFO root [0x00000c20] [2019-03-02 10:16:41,263] - 	componentID : 7001
  WARN root [0x00000c20] [2019-03-02 10:16:41,590] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [0x00000c20] [2019-03-02 10:16:41,803] - Components::findLogger: found logger, addr:192.168.0.123:2851
  INFO root [0x00000c20] [2019-03-02 10:16:41,803] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [0x00000c20] [2019-03-02 10:16:41,803] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00000c20] [2019-03-02 10:16:41,803] - ---- baseapp Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: 18914A7099A0127BD73DD63149F1D9BA. Config: Debug X86. Built: 17:20:50 Feb 27 2019. AppID: 7001. UID: 28936. PID: 12736 ----
  INFO root [0x00000c20] [2019-03-02 10:16:41,803] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [0x00000c20] [2019-03-02 10:16:41,803] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00000c20] [2019-03-02 10:16:41,809] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x00000c20] [2019-03-02 10:16:41,990] - Script::install(): is successfully, Python=(3.7.2 (default, Feb 27 2019, 17:13:46) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x00000c20] [2019-03-02 10:16:41,990] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00000c20] [2019-03-02 10:16:42,082] - loaded entity-script:Account(1).
 DEBUG root [0x00000c20] [2019-03-02 10:16:42,082] - loaded entity-script:Halls(2).
 DEBUG root [0x00000c20] [2019-03-02 10:16:42,082] - loaded entity-script:Room(3).
  INFO root [0x00000c20] [2019-03-02 10:16:42,082] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00000c20] [2019-03-02 10:16:42,082] - TelnetServer server is running on port 40000
  INFO root [0x00000c20] [2019-03-02 10:16:42,433] - ---- baseapp is running ----
 DEBUG root [0x00000c20] [2019-03-02 10:16:42,433] - Components::process(): Request for the process of identity...
  WARN root [0x00000c20] [2019-03-02 10:16:48,633] - Forwarding logs to logger[192.168.0.123:2851/0/0/0]...
 ERROR root [0x00000c20] [2019-03-02 10:17:07,851] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [0x00000c20] [2019-03-02 10:17:07,851] - The following logs sent to logger failed:
  WARN root [0x00000c20] [2019-03-02 10:17:07,851] - ==> [2019-03-02 10:17:07 854] TCPPacketReceiver::processPendingEvents(192.168.0.123:2851): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00000c20] [2019-03-02 10:17:07,851] - ==> [2019-03-02 10:17:07 855] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [0x00000c20] [2019-03-02 10:17:07,878] - TCPPacketReceiver::processPendingEvents(192.168.0.123:2850): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00000c20] [2019-03-02 10:17:07,878] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [0x00000c20] [2019-03-02 10:17:07,888] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00000c20] [2019-03-02 10:17:07,888] - onBaseAppShutDown: state=0
  INFO root [0x00000c20] [2019-03-02 10:17:07,888] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00000c20] [2019-03-02 10:17:07,888] - onBaseAppShutDown: state=1
  INFO root [0x00000c20] [2019-03-02 10:17:07,888] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00000c20] [2019-03-02 10:17:07,888] - onBaseAppShutDown: state=2
 DEBUG root [0x00000c20] [2019-03-02 10:17:07,909] - Halls::~Halls(): 1
  INFO root [0x00000c20] [2019-03-02 10:17:07,922] - Script::uninstall(): is successfully!
 DEBUG root [0x00000c20] [2019-03-02 10:17:07,922] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00002768] [2019-03-02 10:17:15,719] - 
  INFO root [0x00002768] [2019-03-02 10:17:15,719] - -----------------------------------------------------------------------------------------


  INFO root [0x00002768] [2019-03-02 10:17:15,719] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [0x00002768] [2019-03-02 10:17:15,719] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [0x00002768] [2019-03-02 10:17:15,719] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [0x00002768] [2019-03-02 10:17:15,719] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:20015 )
  INFO root [0x00002768] [2019-03-02 10:17:15,719] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.0.123:20015, SOMAXCONN=128.
  INFO root [0x00002768] [2019-03-02 10:17:15,728] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:20005 )
  INFO root [0x00002768] [2019-03-02 10:17:15,728] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.0.123:20005, SOMAXCONN=128.
  INFO root [0x00002768] [2019-03-02 10:17:15,732] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:2900 )
  INFO root [0x00002768] [2019-03-02 10:17:15,732] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.123:2900, SOMAXCONN=128.
  INFO root [0x00002768] [2019-03-02 10:17:15,732] - server-configs:
  INFO root [0x00002768] [2019-03-02 10:17:15,732] - 	gameUpdateHertz : 10
  INFO root [0x00002768] [2019-03-02 10:17:15,732] - 	entryScriptFile : kbemain
  INFO root [0x00002768] [2019-03-02 10:17:15,732] - 	internalTcpAddr : 192.168.0.123:2900
  INFO root [0x00002768] [2019-03-02 10:17:15,732] - 	externalTcpAddr : 192.168.0.123:20015
  INFO root [0x00002768] [2019-03-02 10:17:15,732] - 	externalUdpAddr : 192.168.0.123:20005
  INFO root [0x00002768] [2019-03-02 10:17:15,732] - 	componentID : 7001
  WARN root [0x00002768] [2019-03-02 10:17:16,080] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [0x00002768] [2019-03-02 10:17:16,179] - Components::findLogger: found logger, addr:192.168.0.123:2899
  INFO root [0x00002768] [2019-03-02 10:17:16,181] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [0x00002768] [2019-03-02 10:17:16,181] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002768] [2019-03-02 10:17:16,181] - ---- baseapp Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: 18914A7099A0127BD73DD63149F1D9BA. Config: Debug X86. Built: 17:20:50 Feb 27 2019. AppID: 7001. UID: 28936. PID: 12432 ----
  INFO root [0x00002768] [2019-03-02 10:17:16,181] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [0x00002768] [2019-03-02 10:17:16,181] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002768] [2019-03-02 10:17:16,181] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x00002768] [2019-03-02 10:17:16,368] - Script::install(): is successfully, Python=(3.7.2 (default, Feb 27 2019, 17:13:46) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x00002768] [2019-03-02 10:17:16,368] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00002768] [2019-03-02 10:17:16,468] - loaded entity-script:Account(1).
 DEBUG root [0x00002768] [2019-03-02 10:17:16,468] - loaded entity-script:Halls(2).
 DEBUG root [0x00002768] [2019-03-02 10:17:16,468] - loaded entity-script:Room(3).
  INFO root [0x00002768] [2019-03-02 10:17:16,468] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00002768] [2019-03-02 10:17:16,468] - TelnetServer server is running on port 40000
  INFO root [0x00002768] [2019-03-02 10:17:16,902] - ---- baseapp is running ----
 DEBUG root [0x00002768] [2019-03-02 10:17:16,902] - Components::process(): Request for the process of identity...
  WARN root [0x00002768] [2019-03-02 10:17:23,034] - Forwarding logs to logger[192.168.0.123:2899/0/0/0]...
 ERROR root [0x00002768] [2019-03-02 10:18:33,693] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [0x00002768] [2019-03-02 10:18:33,696] - The following logs sent to logger failed:
  WARN root [0x00002768] [2019-03-02 10:18:33,697] - ==> [2019-03-02 10:18:33 694] TCPPacketReceiver::processPendingEvents(192.168.0.123:2899): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002768] [2019-03-02 10:18:33,697] - ==> [2019-03-02 10:18:33 695] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [0x00002768] [2019-03-02 10:18:33,731] - TCPPacketReceiver::processPendingEvents(192.168.0.123:2902): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002768] [2019-03-02 10:18:33,734] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [0x00002768] [2019-03-02 10:18:33,740] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00002768] [2019-03-02 10:18:33,743] - onBaseAppShutDown: state=0
  INFO root [0x00002768] [2019-03-02 10:18:33,743] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00002768] [2019-03-02 10:18:33,744] - onBaseAppShutDown: state=1
  INFO root [0x00002768] [2019-03-02 10:18:33,744] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00002768] [2019-03-02 10:18:33,744] - onBaseAppShutDown: state=2
 DEBUG root [0x00002768] [2019-03-02 10:18:33,757] - Halls::~Halls(): 1
  INFO root [0x00002768] [2019-03-02 10:18:33,766] - Script::uninstall(): is successfully!
 DEBUG root [0x00002768] [2019-03-02 10:18:33,766] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00001f04] [2019-03-02 10:20:36,608] - 
  INFO root [0x00001f04] [2019-03-02 10:20:36,612] - -----------------------------------------------------------------------------------------


  INFO root [0x00001f04] [2019-03-02 10:20:36,612] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [0x00001f04] [2019-03-02 10:20:36,612] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [0x00001f04] [2019-03-02 10:20:36,612] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [0x00001f04] [2019-03-02 10:20:36,618] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:20015 )
  INFO root [0x00001f04] [2019-03-02 10:20:36,618] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.0.123:20015, SOMAXCONN=128.
  INFO root [0x00001f04] [2019-03-02 10:20:36,622] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:20005 )
  INFO root [0x00001f04] [2019-03-02 10:20:36,628] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.0.123:20005, SOMAXCONN=128.
  INFO root [0x00001f04] [2019-03-02 10:20:36,632] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:2991 )
  INFO root [0x00001f04] [2019-03-02 10:20:36,632] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.123:2991, SOMAXCONN=128.
  INFO root [0x00001f04] [2019-03-02 10:20:36,632] - server-configs:
  INFO root [0x00001f04] [2019-03-02 10:20:36,632] - 	gameUpdateHertz : 10
  INFO root [0x00001f04] [2019-03-02 10:20:36,632] - 	entryScriptFile : kbemain
  INFO root [0x00001f04] [2019-03-02 10:20:36,632] - 	internalTcpAddr : 192.168.0.123:2991
  INFO root [0x00001f04] [2019-03-02 10:20:36,632] - 	externalTcpAddr : 192.168.0.123:20015
  INFO root [0x00001f04] [2019-03-02 10:20:36,632] - 	externalUdpAddr : 192.168.0.123:20005
  INFO root [0x00001f04] [2019-03-02 10:20:36,632] - 	componentID : 7001
  WARN root [0x00001f04] [2019-03-02 10:20:36,918] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [0x00001f04] [2019-03-02 10:20:37,141] - Components::findLogger: found logger, addr:192.168.0.123:2993
  INFO root [0x00001f04] [2019-03-02 10:20:37,141] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [0x00001f04] [2019-03-02 10:20:37,141] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001f04] [2019-03-02 10:20:37,151] - ---- baseapp Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: 18914A7099A0127BD73DD63149F1D9BA. Config: Debug X86. Built: 17:20:50 Feb 27 2019. AppID: 7001. UID: 28936. PID: 8076 ----
  INFO root [0x00001f04] [2019-03-02 10:20:37,151] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [0x00001f04] [2019-03-02 10:20:37,151] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001f04] [2019-03-02 10:20:37,151] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x00001f04] [2019-03-02 10:20:37,332] - Script::install(): is successfully, Python=(3.7.2 (default, Feb 27 2019, 17:13:46) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x00001f04] [2019-03-02 10:20:37,332] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001f04] [2019-03-02 10:20:37,422] - loaded entity-script:Account(1).
 DEBUG root [0x00001f04] [2019-03-02 10:20:37,422] - loaded entity-script:Halls(2).
 DEBUG root [0x00001f04] [2019-03-02 10:20:37,422] - loaded entity-script:Room(3).
  INFO root [0x00001f04] [2019-03-02 10:20:37,422] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001f04] [2019-03-02 10:20:37,428] - TelnetServer server is running on port 40000
  INFO root [0x00001f04] [2019-03-02 10:20:37,907] - ---- baseapp is running ----
 DEBUG root [0x00001f04] [2019-03-02 10:20:37,912] - Components::process(): Request for the process of identity...
  WARN root [0x00001f04] [2019-03-02 10:20:44,088] - Forwarding logs to logger[192.168.0.123:2993/0/0/0]...
 ERROR root [0x00001f04] [2019-03-02 10:27:41,787] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [0x00001f04] [2019-03-02 10:27:41,787] - The following logs sent to logger failed:
  WARN root [0x00001f04] [2019-03-02 10:27:41,787] - ==> [2019-03-02 10:27:41 794] TCPPacketReceiver::processPendingEvents(192.168.0.123:2993): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001f04] [2019-03-02 10:27:41,787] - ==> [2019-03-02 10:27:41 794] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [0x00001f04] [2019-03-02 10:27:41,817] - TCPPacketReceiver::processPendingEvents(192.168.0.123:2988): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001f04] [2019-03-02 10:27:41,817] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [0x00001f04] [2019-03-02 10:27:41,827] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00001f04] [2019-03-02 10:27:41,827] - onBaseAppShutDown: state=0
  INFO root [0x00001f04] [2019-03-02 10:27:41,827] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00001f04] [2019-03-02 10:27:41,827] - onBaseAppShutDown: state=1
  INFO root [0x00001f04] [2019-03-02 10:27:41,827] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00001f04] [2019-03-02 10:27:41,827] - onBaseAppShutDown: state=2
 ERROR root [0x00001f04] [2019-03-02 10:27:41,847] - Account::onCellWriteToDBCompleted(2): not found dbmgr!
 ERROR root [0x00001f04] [2019-03-02 10:27:41,847] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00001f04] [2019-03-02 10:27:41,847] - Account::onCellWriteToDBCompleted(4): not found dbmgr!
 ERROR root [0x00001f04] [2019-03-02 10:27:41,847] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x00001f04] [2019-03-02 10:27:41,857] - Halls::~Halls(): 1
 DEBUG root [0x00001f04] [2019-03-02 10:27:41,857] - Account::~Account(): 2
 DEBUG root [0x00001f04] [2019-03-02 10:27:41,857] - Account::~Account(): 4
 DEBUG root [0x00001f04] [2019-03-02 10:27:41,857] - Room::~Room(): 3
  INFO root [0x00001f04] [2019-03-02 10:27:41,867] - Script::uninstall(): is successfully!
 DEBUG root [0x00001f04] [2019-03-02 10:27:41,867] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00002ad8] [2019-03-02 10:35:36,399] - 
  INFO root [0x00002ad8] [2019-03-02 10:35:36,399] - -----------------------------------------------------------------------------------------


  INFO root [0x00002ad8] [2019-03-02 10:35:36,399] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [0x00002ad8] [2019-03-02 10:35:36,399] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [0x00002ad8] [2019-03-02 10:35:36,399] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [0x00002ad8] [2019-03-02 10:35:36,407] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:20015 )
  INFO root [0x00002ad8] [2019-03-02 10:35:36,407] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.0.123:20015, SOMAXCONN=128.
  INFO root [0x00002ad8] [2019-03-02 10:35:36,414] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:20005 )
  INFO root [0x00002ad8] [2019-03-02 10:35:36,414] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.0.123:20005, SOMAXCONN=128.
  INFO root [0x00002ad8] [2019-03-02 10:35:36,419] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:3252 )
  INFO root [0x00002ad8] [2019-03-02 10:35:36,419] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.123:3252, SOMAXCONN=128.
  INFO root [0x00002ad8] [2019-03-02 10:35:36,419] - server-configs:
  INFO root [0x00002ad8] [2019-03-02 10:35:36,419] - 	gameUpdateHertz : 10
  INFO root [0x00002ad8] [2019-03-02 10:35:36,419] - 	entryScriptFile : kbemain
  INFO root [0x00002ad8] [2019-03-02 10:35:36,419] - 	internalTcpAddr : 192.168.0.123:3252
  INFO root [0x00002ad8] [2019-03-02 10:35:36,419] - 	externalTcpAddr : 192.168.0.123:20015
  INFO root [0x00002ad8] [2019-03-02 10:35:36,419] - 	externalUdpAddr : 192.168.0.123:20005
  INFO root [0x00002ad8] [2019-03-02 10:35:36,419] - 	componentID : 7001
  WARN root [0x00002ad8] [2019-03-02 10:35:36,775] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [0x00002ad8] [2019-03-02 10:35:36,989] - Components::findLogger: found logger, addr:192.168.0.123:3254
  INFO root [0x00002ad8] [2019-03-02 10:35:36,989] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [0x00002ad8] [2019-03-02 10:35:36,989] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002ad8] [2019-03-02 10:35:36,997] - ---- baseapp Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: 18914A7099A0127BD73DD63149F1D9BA. Config: Debug X86. Built: 17:20:50 Feb 27 2019. AppID: 7001. UID: 28936. PID: 7200 ----
  INFO root [0x00002ad8] [2019-03-02 10:35:36,997] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [0x00002ad8] [2019-03-02 10:35:36,997] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002ad8] [2019-03-02 10:35:36,997] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x00002ad8] [2019-03-02 10:35:37,186] - Script::install(): is successfully, Python=(3.7.2 (default, Feb 27 2019, 17:13:46) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x00002ad8] [2019-03-02 10:35:37,186] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00002ad8] [2019-03-02 10:35:37,299] - loaded entity-script:Account(1).
 DEBUG root [0x00002ad8] [2019-03-02 10:35:37,305] - loaded entity-script:Halls(2).
 DEBUG root [0x00002ad8] [2019-03-02 10:35:37,305] - loaded entity-script:Room(3).
  INFO root [0x00002ad8] [2019-03-02 10:35:37,305] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00002ad8] [2019-03-02 10:35:37,305] - TelnetServer server is running on port 40000
  INFO root [0x00002ad8] [2019-03-02 10:35:37,729] - ---- baseapp is running ----
 DEBUG root [0x00002ad8] [2019-03-02 10:35:37,729] - Components::process(): Request for the process of identity...
  WARN root [0x00002ad8] [2019-03-02 10:35:44,006] - Forwarding logs to logger[192.168.0.123:3254/0/0/0]...
 ERROR root [0x00002ad8] [2019-03-02 10:46:59,094] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [0x00002ad8] [2019-03-02 10:46:59,094] - The following logs sent to logger failed:
  WARN root [0x00002ad8] [2019-03-02 10:46:59,104] - ==> [2019-03-02 10:46:59 099] TCPPacketReceiver::processPendingEvents(192.168.0.123:3254): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002ad8] [2019-03-02 10:46:59,104] - ==> [2019-03-02 10:46:59 100] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [0x00002ad8] [2019-03-02 10:46:59,144] - TCPPacketReceiver::processPendingEvents(192.168.0.123:3251): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002ad8] [2019-03-02 10:46:59,144] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [0x00002ad8] [2019-03-02 10:46:59,144] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00002ad8] [2019-03-02 10:46:59,144] - onBaseAppShutDown: state=0
  INFO root [0x00002ad8] [2019-03-02 10:46:59,144] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00002ad8] [2019-03-02 10:46:59,144] - onBaseAppShutDown: state=1
  INFO root [0x00002ad8] [2019-03-02 10:46:59,155] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00002ad8] [2019-03-02 10:46:59,155] - onBaseAppShutDown: state=2
 ERROR root [0x00002ad8] [2019-03-02 10:46:59,165] - Account::onCellWriteToDBCompleted(2): not found dbmgr!
 ERROR root [0x00002ad8] [2019-03-02 10:46:59,165] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00002ad8] [2019-03-02 10:46:59,175] - Account::onCellWriteToDBCompleted(4): not found dbmgr!
 ERROR root [0x00002ad8] [2019-03-02 10:46:59,175] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00002ad8] [2019-03-02 10:46:59,175] - Account::onCellWriteToDBCompleted(5): not found dbmgr!
 ERROR root [0x00002ad8] [2019-03-02 10:46:59,175] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00002ad8] [2019-03-02 10:46:59,175] - Account::onCellWriteToDBCompleted(7): not found dbmgr!
 ERROR root [0x00002ad8] [2019-03-02 10:46:59,175] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00002ad8] [2019-03-02 10:46:59,175] - Account::onCellWriteToDBCompleted(8): not found dbmgr!
 ERROR root [0x00002ad8] [2019-03-02 10:46:59,175] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00002ad8] [2019-03-02 10:46:59,175] - Account::onCellWriteToDBCompleted(10): not found dbmgr!
 ERROR root [0x00002ad8] [2019-03-02 10:46:59,185] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x00002ad8] [2019-03-02 10:46:59,185] - Halls::~Halls(): 1
 DEBUG root [0x00002ad8] [2019-03-02 10:46:59,185] - Account::~Account(): 2
 DEBUG root [0x00002ad8] [2019-03-02 10:46:59,185] - Account::~Account(): 4
 DEBUG root [0x00002ad8] [2019-03-02 10:46:59,185] - Room::~Room(): 3
 DEBUG root [0x00002ad8] [2019-03-02 10:46:59,185] - Account::~Account(): 5
 DEBUG root [0x00002ad8] [2019-03-02 10:46:59,185] - Account::~Account(): 7
 DEBUG root [0x00002ad8] [2019-03-02 10:46:59,185] - Room::~Room(): 6
 DEBUG root [0x00002ad8] [2019-03-02 10:46:59,185] - Account::~Account(): 8
 DEBUG root [0x00002ad8] [2019-03-02 10:46:59,185] - Account::~Account(): 10
 DEBUG root [0x00002ad8] [2019-03-02 10:46:59,187] - Room::~Room(): 9
  INFO root [0x00002ad8] [2019-03-02 10:46:59,193] - Script::uninstall(): is successfully!
 DEBUG root [0x00002ad8] [2019-03-02 10:46:59,193] - ThreadPool::destroy(): starting size 2.
  INFO root [0x0000112c] [2019-03-02 10:48:41,825] - 
  INFO root [0x0000112c] [2019-03-02 10:48:41,825] - -----------------------------------------------------------------------------------------


  INFO root [0x0000112c] [2019-03-02 10:48:41,825] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [0x0000112c] [2019-03-02 10:48:41,834] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [0x0000112c] [2019-03-02 10:48:41,834] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [0x0000112c] [2019-03-02 10:48:41,838] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:20015 )
  INFO root [0x0000112c] [2019-03-02 10:48:41,838] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.0.123:20015, SOMAXCONN=128.
  INFO root [0x0000112c] [2019-03-02 10:48:41,838] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:20005 )
  INFO root [0x0000112c] [2019-03-02 10:48:41,844] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.0.123:20005, SOMAXCONN=128.
  INFO root [0x0000112c] [2019-03-02 10:48:41,844] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:3608 )
  INFO root [0x0000112c] [2019-03-02 10:48:41,844] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.123:3608, SOMAXCONN=128.
  INFO root [0x0000112c] [2019-03-02 10:48:41,844] - server-configs:
  INFO root [0x0000112c] [2019-03-02 10:48:41,844] - 	gameUpdateHertz : 10
  INFO root [0x0000112c] [2019-03-02 10:48:41,844] - 	entryScriptFile : kbemain
  INFO root [0x0000112c] [2019-03-02 10:48:41,844] - 	internalTcpAddr : 192.168.0.123:3608
  INFO root [0x0000112c] [2019-03-02 10:48:41,844] - 	externalTcpAddr : 192.168.0.123:20015
  INFO root [0x0000112c] [2019-03-02 10:48:41,844] - 	externalUdpAddr : 192.168.0.123:20005
  INFO root [0x0000112c] [2019-03-02 10:48:41,844] - 	componentID : 7001
  WARN root [0x0000112c] [2019-03-02 10:48:42,154] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [0x0000112c] [2019-03-02 10:48:42,314] - Components::findLogger: found logger, addr:192.168.0.123:3603
  INFO root [0x0000112c] [2019-03-02 10:48:42,314] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [0x0000112c] [2019-03-02 10:48:42,314] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x0000112c] [2019-03-02 10:48:42,314] - ---- baseapp Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: 18914A7099A0127BD73DD63149F1D9BA. Config: Debug X86. Built: 17:20:50 Feb 27 2019. AppID: 7001. UID: 28936. PID: 2256 ----
  INFO root [0x0000112c] [2019-03-02 10:48:42,314] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [0x0000112c] [2019-03-02 10:48:42,314] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x0000112c] [2019-03-02 10:48:42,314] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x0000112c] [2019-03-02 10:48:42,514] - Script::install(): is successfully, Python=(3.7.2 (default, Feb 27 2019, 17:13:46) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x0000112c] [2019-03-02 10:48:42,514] - Script::install(): KBExtra is successfully!
 DEBUG root [0x0000112c] [2019-03-02 10:48:42,614] - loaded entity-script:Account(1).
 DEBUG root [0x0000112c] [2019-03-02 10:48:42,614] - loaded entity-script:Halls(2).
 DEBUG root [0x0000112c] [2019-03-02 10:48:42,614] - loaded entity-script:Room(3).
  INFO root [0x0000112c] [2019-03-02 10:48:42,614] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x0000112c] [2019-03-02 10:48:42,614] - TelnetServer server is running on port 40000
  INFO root [0x0000112c] [2019-03-02 10:48:42,988] - ---- baseapp is running ----
 DEBUG root [0x0000112c] [2019-03-02 10:48:42,988] - Components::process(): Request for the process of identity...
  WARN root [0x0000112c] [2019-03-02 10:48:49,155] - Forwarding logs to logger[192.168.0.123:3603/0/0/0]...
 ERROR root [0x0000112c] [2019-03-02 11:24:21,571] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [0x0000112c] [2019-03-02 11:24:21,576] - The following logs sent to logger failed:
  WARN root [0x0000112c] [2019-03-02 11:24:21,577] - ==> [2019-03-02 11:24:21 570] TCPPacketReceiver::processPendingEvents(192.168.0.123:3603): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x0000112c] [2019-03-02 11:24:21,577] - ==> [2019-03-02 11:24:21 573] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [0x0000112c] [2019-03-02 11:24:21,648] - TCPPacketReceiver::processPendingEvents(192.168.0.123:3606): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x0000112c] [2019-03-02 11:24:21,649] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [0x0000112c] [2019-03-02 11:24:21,649] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x0000112c] [2019-03-02 11:24:21,650] - onBaseAppShutDown: state=0
  INFO root [0x0000112c] [2019-03-02 11:24:21,650] - Shutdowner::onShutdown: shutting down
S_INFO root [0x0000112c] [2019-03-02 11:24:21,650] - onBaseAppShutDown: state=1
  INFO root [0x0000112c] [2019-03-02 11:24:21,651] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x0000112c] [2019-03-02 11:24:21,651] - onBaseAppShutDown: state=2
 ERROR root [0x0000112c] [2019-03-02 11:24:21,674] - Account::onCellWriteToDBCompleted(2): not found dbmgr!
 ERROR root [0x0000112c] [2019-03-02 11:24:21,675] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x0000112c] [2019-03-02 11:24:21,676] - Account::onCellWriteToDBCompleted(4): not found dbmgr!
 ERROR root [0x0000112c] [2019-03-02 11:24:21,676] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x0000112c] [2019-03-02 11:24:21,679] - Account::onCellWriteToDBCompleted(5): not found dbmgr!
 ERROR root [0x0000112c] [2019-03-02 11:24:21,680] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x0000112c] [2019-03-02 11:24:21,682] - Account::onCellWriteToDBCompleted(7): not found dbmgr!
 ERROR root [0x0000112c] [2019-03-02 11:24:21,682] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x0000112c] [2019-03-02 11:24:21,684] - Account::onCellWriteToDBCompleted(8): not found dbmgr!
 ERROR root [0x0000112c] [2019-03-02 11:24:21,684] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x0000112c] [2019-03-02 11:24:21,685] - Account::onCellWriteToDBCompleted(10): not found dbmgr!
 ERROR root [0x0000112c] [2019-03-02 11:24:21,686] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x0000112c] [2019-03-02 11:24:21,686] - Halls::~Halls(): 1
 DEBUG root [0x0000112c] [2019-03-02 11:24:21,687] - Account::~Account(): 2
 DEBUG root [0x0000112c] [2019-03-02 11:24:21,687] - Account::~Account(): 4
 DEBUG root [0x0000112c] [2019-03-02 11:24:21,687] - Room::~Room(): 3
 DEBUG root [0x0000112c] [2019-03-02 11:24:21,688] - Account::~Account(): 5
 DEBUG root [0x0000112c] [2019-03-02 11:24:21,688] - Account::~Account(): 7
 DEBUG root [0x0000112c] [2019-03-02 11:24:21,688] - Room::~Room(): 6
 DEBUG root [0x0000112c] [2019-03-02 11:24:21,688] - Account::~Account(): 8
 DEBUG root [0x0000112c] [2019-03-02 11:24:21,689] - Account::~Account(): 10
 DEBUG root [0x0000112c] [2019-03-02 11:24:21,689] - Room::~Room(): 9
  INFO root [0x0000112c] [2019-03-02 11:24:21,709] - Script::uninstall(): is successfully!
 DEBUG root [0x0000112c] [2019-03-02 11:24:21,710] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00000614] [2019-03-02 14:44:45,131] - 
  INFO root [0x00000614] [2019-03-02 14:44:45,144] - -----------------------------------------------------------------------------------------


  INFO root [0x00000614] [2019-03-02 14:44:45,176] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [0x00000614] [2019-03-02 14:44:45,176] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [0x00000614] [2019-03-02 14:44:45,176] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [0x00000614] [2019-03-02 14:44:45,246] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:20015 )
  INFO root [0x00000614] [2019-03-02 14:44:45,246] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.0.148:20015, SOMAXCONN=128.
  INFO root [0x00000614] [2019-03-02 14:44:45,250] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:20005 )
  INFO root [0x00000614] [2019-03-02 14:44:45,251] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.0.148:20005, SOMAXCONN=128.
  INFO root [0x00000614] [2019-03-02 14:44:45,254] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:13656 )
  INFO root [0x00000614] [2019-03-02 14:44:45,255] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.148:13656, SOMAXCONN=128.
  INFO root [0x00000614] [2019-03-02 14:44:45,255] - server-configs:
  INFO root [0x00000614] [2019-03-02 14:44:45,256] - 	gameUpdateHertz : 10
  INFO root [0x00000614] [2019-03-02 14:44:45,256] - 	entryScriptFile : kbemain
  INFO root [0x00000614] [2019-03-02 14:44:45,256] - 	internalTcpAddr : 192.168.0.148:13656
  INFO root [0x00000614] [2019-03-02 14:44:45,256] - 	externalTcpAddr : 192.168.0.148:20015
  INFO root [0x00000614] [2019-03-02 14:44:45,257] - 	externalUdpAddr : 192.168.0.148:20005
  INFO root [0x00000614] [2019-03-02 14:44:45,257] - 	componentID : 7001
  WARN root [0x00000614] [2019-03-02 14:44:45,699] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [0x00000614] [2019-03-02 14:44:45,807] - Components::findLogger: found logger, addr:192.168.0.148:13513
  INFO root [0x00000614] [2019-03-02 14:44:45,820] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [0x00000614] [2019-03-02 14:44:45,820] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00000614] [2019-03-02 14:44:45,857] - ---- baseapp Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: 18914A7099A0127BD73DD63149F1D9BA. Config: Debug X86. Built: 17:20:50 Feb 27 2019. AppID: 7001. UID: 28936. PID: 2652 ----
  INFO root [0x00000614] [2019-03-02 14:44:45,858] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [0x00000614] [2019-03-02 14:44:45,859] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00000614] [2019-03-02 14:44:45,869] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x00000614] [2019-03-02 14:44:46,091] - Script::install(): is successfully, Python=(3.7.2 (default, Feb 27 2019, 17:13:46) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x00000614] [2019-03-02 14:44:46,091] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00000614] [2019-03-02 14:44:46,903] - loaded entity-script:Account(1).
 DEBUG root [0x00000614] [2019-03-02 14:44:46,904] - loaded entity-script:Halls(2).
 DEBUG root [0x00000614] [2019-03-02 14:44:46,904] - loaded entity-script:Room(3).
  INFO root [0x00000614] [2019-03-02 14:44:46,916] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00000614] [2019-03-02 14:44:46,942] - TelnetServer server is running on port 40000
  INFO root [0x00000614] [2019-03-02 14:44:47,709] - ---- baseapp is running ----
 DEBUG root [0x00000614] [2019-03-02 14:44:47,710] - Components::process(): Request for the process of identity...
  WARN root [0x00000614] [2019-03-02 14:44:53,960] - Forwarding logs to logger[192.168.0.148:13513/0/0/0]...
 ERROR root [0x00000614] [2019-03-02 14:45:30,902] - Components::removeComponentByChannel: baseappmgr : 5000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [0x00000614] [2019-03-02 14:45:31,005] - The following logs sent to logger failed:
  WARN root [0x00000614] [2019-03-02 14:45:31,012] - ==> [2019-03-02 14:45:31 005] TCPPacketReceiver::processPendingEvents(192.168.0.148:13513): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [0x00000614] [2019-03-02 14:45:31,013] - ==> [2019-03-02 14:45:31 006] Components::removeComponentByChannel: logger : 2000, Normal exit!
  WARN root [0x00000614] [2019-03-02 14:45:31,013] - TCPPacketReceiver::processPendingEvents(192.168.0.148:13566): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [0x00000614] [2019-03-02 14:45:31,014] - Components::removeComponentByChannel: dbmgr : 4000, Normal exit!
  WARN root [0x00000614] [2019-03-02 14:45:31,014] - ServerApp::shutDown:  In shuttingdown!
  WARN root [0x00000614] [2019-03-02 14:45:31,015] - TCPPacketReceiver::processPendingEvents(192.168.0.148:13637): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [0x00000614] [2019-03-02 14:45:31,015] - Components::removeComponentByChannel: cellappmgr : 6000, Normal exit!
  INFO root [0x00000614] [2019-03-02 14:45:31,966] - Shutdowner::onShutdown: shutting down
  INFO root [0x00000614] [2019-03-02 14:45:31,966] - Baseapp::canShutdown(): Waiting for cellapp[8001, ] destruction!
  INFO root [0x00000614] [2019-03-02 14:45:32,763] - Components::removeComponentByChannel: cellapp : 8001, Normal exit!
S_INFO root [0x00000614] [2019-03-02 14:45:32,968] - onReadyForShutDown()
  INFO root [0x00000614] [2019-03-02 14:45:32,968] - Baseapp::canShutdown(): Wait for the entity's into the database! The remaining 1.
S_INFO root [0x00000614] [2019-03-02 14:45:32,968] - onBaseAppShutDown: state=1
S_INFO root [0x00000614] [2019-03-02 14:45:33,989] - onReadyForShutDown()
  INFO root [0x00000614] [2019-03-02 14:45:33,989] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00000614] [2019-03-02 14:45:34,090] - onBaseAppShutDown: state=2
 DEBUG root [0x00000614] [2019-03-02 14:45:34,117] - Halls::~Halls(): 1
  INFO root [0x00000614] [2019-03-02 14:45:34,143] - Script::uninstall(): is successfully!
 DEBUG root [0x00000614] [2019-03-02 14:45:34,143] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00000614] [2019-03-02 14:45:34,445] - ThreadPool::destroy(): waiting for thread(2)[0x20dcb28,0x20dcfa0,], try=1
 DEBUG root [0x00000614] [2019-03-02 14:45:34,847] - TPThread::~TPThread(): 0x20dcb28
 DEBUG root [0x00000614] [2019-03-02 14:45:34,847] - TPThread::~TPThread(): 0x20dcfa0
 DEBUG root [0x00000614] [2019-03-02 14:45:34,847] - ThreadPool::destroy(): successfully!
 DEBUG root [0x00000614] [2019-03-02 14:45:34,856] - Bundle::destroyObjPool(): size 27.
 DEBUG root [0x00000614] [2019-03-02 14:45:34,856] - Channel::destroyObjPool(): size 16.
 DEBUG root [0x00000614] [2019-03-02 14:45:34,857] - TCPPacket::destroyObjPool(): size 26.
 DEBUG root [0x00000614] [2019-03-02 14:45:34,857] - UDPPacket::destroyObjPool(): size 0.
 DEBUG root [0x00000614] [2019-03-02 14:45:34,857] - EndPoint::destroyObjPool(): size 16.
 DEBUG root [0x00000614] [2019-03-02 14:45:34,857] - Address::destroyObjPool(): size 0.
 DEBUG root [0x00000614] [2019-03-02 14:45:34,858] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [0x00000614] [2019-03-02 14:45:34,860] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [0x00000614] [2019-03-02 14:45:34,861] - baseapp(7001) has shut down.
  INFO root [0x00000614] [2019-03-02 14:45:35,863] - EventDispatcher()::~EventDispatcher: Num timers = 1
  INFO root [0x00002048] [2019-03-02 14:45:57,939] - 
  INFO root [0x00002048] [2019-03-02 14:45:57,949] - -----------------------------------------------------------------------------------------


  INFO root [0x00002048] [2019-03-02 14:45:57,950] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [0x00002048] [2019-03-02 14:45:57,951] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [0x00002048] [2019-03-02 14:45:57,951] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [0x00002048] [2019-03-02 14:45:57,962] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:20015 )
  INFO root [0x00002048] [2019-03-02 14:45:57,963] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.0.148:20015, SOMAXCONN=128.
  INFO root [0x00002048] [2019-03-02 14:45:57,968] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:20005 )
  INFO root [0x00002048] [2019-03-02 14:45:57,968] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.0.148:20005, SOMAXCONN=128.
  INFO root [0x00002048] [2019-03-02 14:45:57,972] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:14065 )
  INFO root [0x00002048] [2019-03-02 14:45:57,973] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.148:14065, SOMAXCONN=128.
  INFO root [0x00002048] [2019-03-02 14:45:57,973] - server-configs:
  INFO root [0x00002048] [2019-03-02 14:45:57,973] - 	gameUpdateHertz : 10
  INFO root [0x00002048] [2019-03-02 14:45:57,974] - 	entryScriptFile : kbemain
  INFO root [0x00002048] [2019-03-02 14:45:57,974] - 	internalTcpAddr : 192.168.0.148:14065
  INFO root [0x00002048] [2019-03-02 14:45:57,974] - 	externalTcpAddr : 192.168.0.148:20015
  INFO root [0x00002048] [2019-03-02 14:45:57,974] - 	externalUdpAddr : 192.168.0.148:20005
  INFO root [0x00002048] [2019-03-02 14:45:57,975] - 	componentID : 7001
  WARN root [0x00002048] [2019-03-02 14:45:58,523] - invalid UID(0) <= 0, please check UID for environment! automatically set to 34643.
  INFO root [0x00002048] [2019-03-02 14:45:58,831] - Components::findLogger: found logger, addr:192.168.0.148:14064
  INFO root [0x00002048] [2019-03-02 14:45:58,831] - Components::addComponent: reset orderLog, uid=34643!
  INFO root [0x00002048] [2019-03-02 14:45:58,831] - Components::addComponent[logger], uid=34643, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002048] [2019-03-02 14:45:58,839] - ---- baseapp Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: 18914A7099A0127BD73DD63149F1D9BA. Config: Debug X86. Built: 17:20:50 Feb 27 2019. AppID: 7001. UID: 34643. PID: 14160 ----
  INFO root [0x00002048] [2019-03-02 14:45:58,843] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [0x00002048] [2019-03-02 14:45:58,844] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002048] [2019-03-02 14:45:58,845] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x00002048] [2019-03-02 14:45:59,117] - Script::install(): is successfully, Python=(3.7.2 (default, Feb 27 2019, 17:13:46) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x00002048] [2019-03-02 14:45:59,118] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00002048] [2019-03-02 14:45:59,331] - loaded entity-script:Account(1).
 DEBUG root [0x00002048] [2019-03-02 14:45:59,332] - loaded entity-script:Halls(2).
 DEBUG root [0x00002048] [2019-03-02 14:45:59,333] - loaded entity-script:Room(3).
  INFO root [0x00002048] [2019-03-02 14:45:59,333] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00002048] [2019-03-02 14:45:59,333] - TelnetServer server is running on port 40000
  INFO root [0x00002048] [2019-03-02 14:46:00,512] - ---- baseapp is running ----
 DEBUG root [0x00002048] [2019-03-02 14:46:00,514] - Components::process(): Request for the process of identity...
  WARN root [0x00002048] [2019-03-02 14:46:06,736] - Forwarding logs to logger[192.168.0.148:14064/0/0/0]...
 ERROR root [0x00002048] [2019-03-02 14:49:22,829] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

 ERROR root [0x00002048] [2019-03-02 14:49:22,855] - Account::onCellWriteToDBCompleted(2): not found dbmgr!
 ERROR root [0x00002048] [2019-03-02 14:49:22,855] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00002048] [2019-03-02 14:49:22,855] - Account::onCellWriteToDBCompleted(4): not found dbmgr!
 ERROR root [0x00002048] [2019-03-02 14:49:22,855] - Entity::onDestroy: not found dbmgr!
